c++/src/RleDecoderV2.cc

```diff
@@ -432,6 +435,7 @@ void RleDecoderV2::skip(uint64_t numValues) {
 void RleDecoderV2::next(int64_t* const data,
                         const uint64_t numValues,
                         const char* const notNull) {
+  DEFINE_AUTO_STOPWATCH(metrics, DecodingLatencyUs, DecodingCount);
```

Comment:
Where is the stop watch call in the RleDecoderV1 ?

Reply:
In addition, all IOs happen in the call of readByte(). We probably should make metrics a member varialbe of the RleDecoder and counts its io time in readByte(). Finally, we can exlucde the io time from decoding latancy here to get the pure RLE decoding time.

Reply:
Currently the pure decoding time is got by subtracting the time taken by the readByte() function.